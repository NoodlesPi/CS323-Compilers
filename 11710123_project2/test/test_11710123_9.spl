int compare(int a){
    return a;
}
float compare(int a){
    return a;
}
int main(){
    int k = 0;
    k.a = 10;
    return compare.m;
}