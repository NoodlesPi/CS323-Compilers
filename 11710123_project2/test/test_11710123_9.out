Error type 8 at Line 5: the function’s return value type mismatches the declared type
Error type 4 at Line 4: function is redefined
Error type 9 at Line 9: the function’s arguments mismatch the declared parameters
Error type 13 at Line 9: accessing member of non-structure variable
Error type 8 at Line 9: the function’s return value type mismatches the declared type
